va/validation-authority_test.go

```diff
@@ -915,8 +894,16 @@ func TestGetCAASetFallback(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to create CAADistributedResolver")
 	caaDR.URI = testSrv.URL
 	caaDR.Clients["1.1.1.1"] = new(http.Client)
-	va := NewValidationAuthorityImpl(&cmd.PortConfig{}, nil, nil, caaDR, stats, clock.Default())
-	va.DNSResolver = &bdns.MockDNSResolver{}
+	va := NewValidationAuthorityImpl(
+		&cmd.PortConfig{},
+		nil,
+		nil,
+		caaDR,
```

Comment:
is it worth adding an additional argument to `setup()` so that this `NewValidationAuthorityImpl()` call can be replaced with `setup(caaDR)`? Alternatively could perhaps use `setup()` as is and explicitly set `va.caaDR = caaDR` since this test is in-package and can use the unexported field? Might be bikeshedding.